build.sbt

```diff
@@ -230,6 +230,7 @@ lazy val sparkBackend = (project in backendRoot / "spark")
 
 lazy val bcsBackend = (project in backendRoot / "bcs")
   .withLibrarySettings("cromwell-bcs-backend", bcsBackendDependencies)
+  .dependsOn(engine)
```

Comment:
Remove this dependency... assuming there's no reason to the `BcsConfiguration` to tap into the engine to search for a hardcoded `"BCS"` backend-config.

Reply:
Removed